### ISSUE TITLE
Redundant `imagetype` argument

### ISSUE THREAD

**Issue by stefaneng** [uclahs-cds/public-R-VennDiagram#24]
If `filename` is specified with an extension then `imagetype` parameter should be inferred automatically. 

Two solutions:

1. Leaving both arguments in I propose that we:
  a. First look for extension in `filename`
  b. If no extension then find extension from `imagetype`
2. Deprecate `imagetype` argument and only access filetype via extension given in `filename`

Current we can create a `tiff` file with a `pdf` extension without any kind of warning:

```r
venn.diagram(
	x = list(A = 1:150, B = 121:170),
	filename = 'test.pdf',
	imagetype = 'tiff',
	disable.logging = TRUE
	)
```

**Comment by pboutros** [comment]
Yup makes sense to deprecate `imagetype`, and potentially to import the `write.plot` function that's in [BPG](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/blob/main/R/write.plot.R)
